src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1594,7 +1595,7 @@ public sealed class StochasticGradientDescentClassificationTrainer :
         internal const string UserNameValue = "Hogwild SGD (binary)";
         internal const string ShortName = "HogwildSGD";
 
-        public sealed class Arguments : LearnerInputBaseWithWeight
+        public sealed class Options : LearnerInputBaseWithWeight
         {
             [Argument(ArgumentType.Multiple, HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
             public ISupportClassificationLossFactory LossFunction = new LogLossFactory();
```

Comment:
>ISupportClassificationLossFactory [](start = 19, length = 33)

XML docs

Reply:
Created #2154  to fix this issue in bulk for several learners

---
In reply to: [248022036](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248022036) [](ancestors = 248022036)